src/mime_header.h

```diff
@@ -11,6 +11,8 @@
 #ifndef SQUID_MIME_HEADER_H_
 #define SQUID_MIME_HEADER_H_
 
+#include "sbuf/SBuf.h"
```

Comment:
We should de-inline headersEnd(SBuf) instead IMO. I do not insist on that though.

Reply:
done (will push shortly)